src/flavors/williams/static/components/input-explorer.js

```diff
@@ -105,6 +108,7 @@ class InputExplorer extends Component {
           communityInput={this.props.communityInput}
           subcategoryNames={this.subcategoryNames}
           visibility={(selectedCategory === constants.INPUT_CATEGORY_SUMMARY_NAME) ? true : false}
+          languageCode={this.props.languageCode}
```

Comment:
I don't see this prop being used in InputExplorerSummary. Why are we passing it into that component?